miniapp/src/main/java/com/rakuten/tech/mobile/miniapp/MiniAppDisplay.kt

```diff
@@ -16,7 +17,17 @@ interface MiniAppDisplay : LifecycleObserver {
      * @return [View] as mini app's view with [LayoutParams] set to match
      * the parent's dimensions.
      */
-    fun getMiniAppView(): View
+    @Deprecated(message = "Please replace with getMiniAppView(Context)")
+    suspend fun getMiniAppView(): View
+
+    /**
+     * Provides the view associated with the mini app to the caller for showing the mini app.
+     * @param providedContext is used by the view for initializing the internal services.
+     * Should be the context of activity to ensure that all standard html components work properly.
+     * @return [View] as mini app's view with [LayoutParams] set to match
+     * the parent's dimensions.
+     */
+    suspend fun getMiniAppView(providedContext: Context): View?
```

Comment:
Let's keep the name of the param explicitly `activityContext`.

Reply:
Hmmm I think the hostapp can provide any context they want, though, `ApplicationContext` causes webview not working properly. They may even provide the context from `Service`.

Reply:
That's why we want to tell the developers explicitly to only provide `activityContext` at this stage, and in SDK we are validating if it is of type `Activity` or `ActivityCompat` otherwise returning null for the new API.

Reply:
We could add in the docs:
`This version of retrieval creates the mini app view which doesn't render some of the native elements of web tech. We recommend, using the updated API.`

Reply:
Updated.